app/components/edit/EditAddress.jsx

```diff
@@ -51,120 +65,140 @@ class EditAddress extends Component {
+              <input
+                id={htmlID}
+                type="text"
+                className="input"
+                placeholder="Name"
+                data-field="name"
+                defaultValue={address.name}
+                onChange={handleAddressChange}
+              />
+              <input
+                id={htmlID}
+                type="text"
+                className="input"
+                placeholder="Address 1"
+                data-field="address_1"
+                defaultValue={address.address_1}
+                onChange={handleAddressChange}
+              />
+              <input
+                id={htmlID}
+                type="text"
+                className="input"
+                placeholder="Address 2"
+                data-field="address_2"
+                defaultValue={address.address_2}
+                onChange={handleAddressChange}
+              />
+              <input
+                id={htmlID}
+                type="text"
+                className="input"
+                placeholder="Address 3"
+                data-field="address_3"
+                defaultValue={address.address_3}
+                onChange={handleAddressChange}
+              />
+              <input
+                id={htmlID}
+                type="text"
+                className="input"
+                placeholder="Address 4"
+                data-field="address_4"
+                defaultValue={address.address_4}
+                onChange={handleAddressChange}
+              />
+              <input
+                id={htmlID}
+                type="text"
+                className="input"
+                placeholder="City"
+                data-field="city"
+                defaultValue={address.city}
+                onChange={handleAddressChange}
+              />
+              <input
+                id={htmlID}
+                type="text"
+                className="input"
+                placeholder="State/Province"
+                data-field="state_province"
+                defaultValue={address.state_province}
+                onChange={handleAddressChange}
+              />
+              <input
+                id={htmlID}
+                type="text"
+                className="input"
+                placeholder="Country"
+                data-field="country"
+                defaultValue={address.country}
+                onChange={handleAddressChange}
+              />
+              <input
+                id={htmlID}
+                type="text"
+                className="input"
+                placeholder="Postal/Zip Code"
+                data-field="postal_code"
+                defaultValue={address.postal_code}
+                onChange={handleAddressChange}
+              />
           </div>
         )
     }
   </li>
 );
 
-export default EditAddress;
+const EditAddresses = editCollectionHOC(EditAddress, 'Addresses', {}, true);
```

Comment:
As soon as I add this (with the intention of reusing the `Add` button from other edit components like the `EditPhone` component, any previously saved address doesn't re-populate on the page.